server/models/default_local_provider.go

```diff
@@ -81,6 +82,10 @@ func (l *DefaultLocalProvider) Initialize() {
 		{Feature: PersistMesheryFilters},
 		{Feature: PersistCredentials},
 	}
+	l.TMPMeshsyncDeploymentMode = "operator"
+	if viper.GetBool("TMP_MESHYNC_AS_A_LIBRARY_MODE") {
+		l.TMPMeshsyncDeploymentMode = "library"
```

Comment:
When we do have a new `mode` property defined in our Connections schema, an enum (or semblance of one) with options of "external" and "embedded" makes sense to me.